Lesson2/src/Homework2.java

```diff
@@ -0,0 +1,94 @@
+public class Homework2 {
+
+    public static void main(String[] args) {
+        System.out.println(sum(10, 20));
+//        System.out.println(sum(Integer.MAX_VALUE, Integer.MAX_VALUE));
+        System.out.println(max(50, 49));
+        System.out.println(average(new int[]{1, 2, 3, 4, 5}));
+//        System.out.println(max(new int[]{1, 2, 3, 4, 5, 100, 99}));
+//        System.out.println(calculateHypotenuse());
+    }
+
+    /**
+     * 1. Method should return sum of "a" and "b"
+     * 2. (optional) if sum of a and b is more then int can store -- return -1;
+     **/
+    public static int sum(int a, int b) {
+
+        return a + b;
+    }
+
+    /**
+     * Method should return larger of two numbers ("a" and "b")
+     * <p>
+     * Example1:
+     * a = 4,
+     * b = 5
+     * <p>
+     * Method should return 5
+     * Example2:
+     * a = 10,
+     * b = 10
+     * <p>
+     * Method should return 10
+     */
+    public static int max(int a, int b) {
+//        int answer;
+//       if (a > b)
+//        {
+//            answer = a;
+//        }
+//        else
+//        {
+//            answer = b;
+//        }
+
+//        answer = Math.max(a,b);
+//        return answer;
+
+        return Math.max(a,b);
+    }
+
+    /**
+     * Method should return avg. value of given array (sum of all elements of array divided by array length).
+     * <p>
+     * Example:
+     * array = {1,2,3,4,5}
+     * Method should return 3
+     */
+    public static double average(int[] array) {
+        double sum = 0;
+        for (int i = 0; i < array.length; i++) {
+            sum = sum + array[i];
+        }
+        return (sum/array.length);
```

Comment:
parentheses are not required here, but they aren't break anything, so not a big deal

please fix code formatting by either pressing ctrl + alt + l or just adding spaces before and after "/" sign. Everything else looks good to me

Are you planning to add some of the optional tasks today?

Reply:
Yes, it's done!